test/Microsoft.ML.StaticPipelineTesting/Microsoft.ML.StaticPipelineTesting.csproj

```diff
@@ -7,6 +7,7 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.ImageAnalytics\Microsoft.ML.ImageAnalytics.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.StandardLearners\Microsoft.ML.StandardLearners.csproj" />
     <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
+    <NativeAssemblyReference Include="FactorizationMachineNative" />
```

Comment:
>FactorizationMachineNative [](start = 38, length = 26)

Hmmm. Opportunistic fix for some other issue?

Reply:
No, it's that FFM requires FFMNative, and I added a test for it in static training

---
In reply to: [219902222](https://github.com/dotnet/machinelearning/pull/993#discussion_r219902222) [](ancestors = 219902222)